### PR TITLE
feature10

### PR DESCRIPTION
**Pull request by jayarajac1**
feature10

### CODE REVIEW

File: ProductService/Controllers/HomeController.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 using System.Web.Mvc;
 
 /// <summary>
-/// 
+/// Home controller
```
Comment (jayarajac) [inline ProductService/Controllers/HomeController.cs]: add descriptive summary

Comment (jayarajac1) [reply ProductService/Controllers/HomeController.cs]: yes

File: ProductService/Controllers/ValuesController.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@
 using System.Web.Http;
 
 /// <summary>
-/// Values Controller provides
+/// Values Controller provides operation on values
 /// </summary>
```
Comment (jayarajac) [inline ProductService/Controllers/ValuesController.cs]: okkjihui

**Comment by jayarajac** [comment]
dsad

File: ProductService/Controllers/HomeController.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@
 using System.Web.Mvc;
 
 /// <summary>
-/// 
+/// Home controller1
```
Comment (jayarajac) [inline ProductService/Controllers/HomeController.cs]: dsafdsafsdfs

File: ProductService/Controllers/HomeController.cs [hunk 4]
```diff
@@ -5,7 +5,7 @@
 using System.Web.Mvc;
 
 /// <summary>
-///
```
Comment (jayarajac1) [inline ProductService/Controllers/HomeController.cs]: dsadsadsa

Comment (jayarajac1) [inline ProductService/Controllers/HomeController.cs]: dsadsad

File: ProductService/Controllers/ValuesController.cs [hunk 5]
```diff
@@ -6,7 +6,7 @@
 using System.Web.Http;
 
 /// <summary>
```
Comment (jayarajac1) [inline ProductService/Controllers/ValuesController.cs]: fdsfsdfsfs

Comment (jayarajac) [inline ProductService/Controllers/ValuesController.cs]: dsafdsfdsfds

**Pull request merged**